docker-compose.override.yml.example

```diff
@@ -40,6 +40,18 @@ services:
     #   - .:/usr/src/app
     # command: sleep infinity
 
+  # Uncomment this service to automatically generate a new block every 5 seconds.
+  # regtest-watcher:
+  #   image: node_poet-node:latest
```

Comment:
if we are to be pulling this image from the Docker registry, we need to specify `poetapp/node:latest`.

Reply:
Since this only is used for local development, the original name used is the correct one.